shared/reports/resources.py

```diff
@@ -424,6 +454,38 @@ def line_without_labels(
             sessions=new_sessions,
         )
 
+    def delete_labels(
```

Comment:
Is it worth documenting somehow that these functionalities used to belong to an EditableReport and that should be used only in particular contexts? I guess we're not making that distinction any more since we're merging Report related classes, so I guess it's fair game to use any property/methods since they're moved to the ReportFile

Reply:
Yes, it should be fair game to use any method now.
In reality, all the mutating methods were already part of the previous interface anyway.